harness/determined/gpu.py

```diff
@@ -109,15 +109,23 @@ def _get_rocm_gpus() -> List[GPU]:
     return gpus
 
 
-def get_gpus() -> List[GPU]:
+def get_gpus() -> Tuple[List[GPU], str]:
```

Comment:
From my perspective, it's a little weird for a function called `get_gpus` to spit out `[], 'cpu'`.  I think having it return `[], None` if this is a CPU-only application makes more sense from an API perspective.

If the `cpu` tag matters elsewhere, then we can keep it as-is and rename this function to something like `get_accelerators` or `get_compute_resources`. But I don't really like this idea as much as just returning None.

Reply:
done!